src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from enum import Enum
+from typing import Dict, List, Optional
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from serenity_types.refdata.options import OptionType
+
+
+class VolModel(Enum):
+    """
+    Currently supported volatility models.
+    """
+
+    SVI = "SVI"
+    """
+    Stochastic volatility (SVI) calibrated volatility model.
+    """
+
+    BLACK_SCHOLES = "BLACK_SCHOLES"
+    """
+    Classic Black-Scholes volatility model.
+    """
+
+
+class VolSurfaceType(Enum):
+    """
+    Representation of the volatility surface, e.g. raw inputs vs. fitted surface.
+    """
+
+    RAW = "RAW"
+    """
+    A volatility surface containing the raw inputs: the option prices, IV's, etc..
+    """
+
+    FITTED = "FITTED"
+    """
+    A calibrated and fitted volatility surface with a dense grid of IV's.
+    """
+
+
+class StrikeType(Enum):
+    """
+    Currently supported strike representations.
+    """
+
+    ABSOLUTE = "ABSOLUTE"
+    """
+    Absolute value of strike, e.g. the 20000 option.
+    """
+
+    LOG_MONEYNESS = "LOG_MONEYNESS"
+    """
+    Relative value of strike vs. current spot, with log transformation, e.g. 1.05
+    """
+
+
+class VolatilitySurface(BaseModel):
+    """
+    Base type for both RAW and INTERPOLATED yield curve representations: a term structure.
+    """
+
```

Comment:
Based on our email discussion, should we add UUID here?

Reply:
Yes, I will add today.